js/navigation/NavigationLayout.ios.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import {
+  StackNavigation,
+  TabNavigation,
+  TabNavigationItem as TabItem,
+} from '@expo/ex-navigation';
+import {
+  Text
+} from 'react-native';
+
+import Router from './routes'
+
+class NavigationLayout extends Component {
+  render() {
+    return (
+      <TabNavigation
+        id="tabMain"
+        initialTab="landing"
+      >
+        <TabItem
+          id="landing"
+          title="landing test"
+          renderTitle={this.renderTitle} 
+        >
+          <StackNavigation
+            id="landing"
+            navigatorUID="landing"
+            initialRoute={Router.getRoute('landing')}
+          />
+        </TabItem>
+      </TabNavigation>  
+    );
+  }
+
+  renderTitle(isSelected, title) {
+    const titleStyle = {
+      color: isSelected ? 'white' : '#999999',
```

Comment:
Encode all colours in a single file, and export when needed. This will make developing the colour theme easier.

Reply:
Will update once owner of globally accessible stylesheet has completed task.